.github/workflows/ci-nightly.yml

```diff
@@ -9,27 +9,6 @@ env:
   CTEST_OUTPUT_ON_FAILURE: 1
 
 jobs:
-  linux:
```

Comment:
This means you also have to remove it from `needs` lower in the list. This workflow is currently invalid and can't be executed :)